haas/ext/switches/migrations/dell/099b939261c1_rename_dell_switch_table_for_flask_.py

```diff
@@ -0,0 +1,27 @@
+"""Rename dell switch table for Flask-SQLAlchemy
```

Comment:
@SahilTikale, it's the same thing as ` haas/migrations/versions/6a8c19565060_move_to_flask.py ` but for the extensions, as opposed to the tables in haas core. That file contains a better explanation. I'll add a cross reference at least.